### DISCUSSION TITLE
Turning 'Adopters' page into 'Case Studies'

### DISCUSSION THREAD

**Discussion by svrnm** [open-telemetry/opentelemetry.io#3813] [Information Architecture]
This is once again an idea how we can improve one of our eco-system pages, this time the adopters pages. 

Why I think the current version is not good:
* The term "adopters" is still confusing, since (a) the [CNCF definition is much broader](https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-adopter) and (b) it's in general hard to distinguish it from "integration" or "being a vendor" (Vendor X adopted OpenTelemetry to ingest telemetry, Library X adopted OpenTelemetry to enable ootb observability)
* Besides only a few of the companies listed we have a lot of names without a way to validate that they use (or ever have used) OpenTelemetry
* The page is boring.

Looking for alternatives, I of course checked what K8s is doing, and I really like the way to present end-users adopting k8s in a case study manner (see https://kubernetes.io/case-studies/) and I think we should be able to evolve the current page easily into something similar:

* Rename "Adopters" to "Case Studies" or "User Stories", etc.
* Take the existing internal blog post (eBay, Farfetch, Uplight) and feature them with a quote at the top of the page
* Take the existing external blog posts (GitHub & Ecobee), note that they are external and feature them as well.
* (Optional) Add some other external resources (kubecon talks etc) onto that same page
* Put the other adopters where we do not have links at the bottom of the page with a few words like "there are many companies worldwide that have adopted OpenTelemetry as end-users, etc. etc"

**Comment by theletterf** [comment]
I like this. It might help to have a landing page with card elements and logos, so as to facilitate navigation and visual recognition. "Oh, look, Splunk!"

**Comment by austinlparker** [comment]
Yeah, getting some formal case studies on there would be good. I would note that I don't necessarily think we can repurpose existing content without rewriting them to be more case study-y though.

**Comment by svrnm** [comment]
> I like this. It might help to have a landing page with card elements and logos, so as to facilitate navigation and visual recognition. "Oh, look, Splunk!"

👍 , also this is for the Adopters page and so far vendors are not listed there, so there will be no "Oh, look, Splunk". Does not mean we can add logos to the vendors page eventually as well (separate discussion!) :-)

> Yeah, getting some formal case studies on there would be good. I would note that I don't necessarily think we can repurpose existing content without rewriting them to be more case study-y though.

Fair point, maybe we start with "User Stories" or something. As said one of my goals is to get this "Adopters" removed somehow and make it more clear that this is about real users of OpenTelemetry vs Integratons vs Vendors (I don't want to say "End-Users", since we have community/end users as well)

**Comment by chalin** [comment]
Interesting idea. FYI, the CNCF has a formal Case Studies page, see:

- https://www.cncf.io/case-studies
- https://www.cncf.io/case-studies?_sft_lf-project=opentelemetry for OTel case studies

If a case study is significant enough, I believe that CNCF resource would be available to help put together a writeup that would be hosted on the CNCF site.

> <img width="1255" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/4140793/db21f79d-9f6a-4168-be2d-e9f37fb8fa97">

**Comment by svrnm** [comment]
> Interesting idea. FYI, the CNCF has a formal Case Studies page, see:
> 
>     * [cncf.io/case-studies](https://www.cncf.io/case-studies)
> 
>     * [cncf.io/case-studies?_sft_lf-project=opentelemetry](https://www.cncf.io/case-studies?_sft_lf-project=opentelemetry) for OTel case studies
> 
> 
> If a case study is significant enough, I believe that CNCF resource would be available to help put together a writeup that would be hosted on the CNCF site.
> 
> > <img alt="image" width="1255" src="https://github.com/open-telemetry/opentelemetry.io/assets/4140793/db21f79d-9f6a-4168-be2d-e9f37fb8fa97">

thanks for sharing that @chalin, it is interesting that there are 3 case studies with OpenTelemetry, but only 1 mentions it in the written text.
